docs/source/help-support.rst

```diff
@@ -1,35 +1,49 @@
-=================
+================
 Help and support
-=================
+================
 
-* Get free community support with our `Google group
-  <https://groups.google.com/a/anaconda.com/forum/#!forum/anaconda>`_.
+Free community support
```

Comment:
```suggestion
Community support
```

I think "Community support" sounds better here. Free sounds like it's some sort of cheap giveaway 😅.

And yes, I understand this may partially awaken the free as in "free pizza"/free as in "freedom" debate! 😂

Reply:
Agreed. I was coming at it with my Anaconda hat on instead of my conda hat. Good callout.